client/playwright.config.ts

```diff
@@ -43,7 +45,11 @@ export default defineConfig({
   projects: [
     {
       name: 'Google Chrome',
-      use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+      use: {
+        ...devices['Desktop Chrome'], ...(
+          // In CI we do not specify the channel to be able to use pre-installed chromium
```

Comment:
There are differences between Chrome and Chromium (something about media codecs), which is why we force Chrome by default.